lib/package.js

```diff
@@ -509,6 +509,13 @@
 
   function loadPackageConfigPath(loader, pkgName, pkgConfigPath) {
     var configLoader = loader.pluginLoader || loader;
+    if (configLoader.execute === false) {
+      var config = configLoader.getConfig();
+      configLoader = new configLoader.constructor();
+      configLoader.config(config);
+      configLoader.execute = true;
+      loader.pluginLoader = configLoader;
+    }
```

Comment:
Wrt the previous comments, I'm not sure we should include this here actually.

The same thing should apply everywhere the pluginLoader is accessed.

Alternatively we can throw a better error that the pluginLoader needs to be set when execution is disabled.